app/models/manageiq/providers/workflows/automation_manager/credential.rb

```diff
@@ -1,4 +1,72 @@
 class ManageIQ::Providers::Workflows::AutomationManager::Credential < ManageIQ::Providers::EmbeddedAutomationManager::Authentication
   validates :ems_ref, :presence => true, :uniqueness_when_changed => {:scope => [:tenant_id]},
             :format => {:with => /\A[\w\-]+\z/i, :message => N_("may contain only alphanumeric and _ - characters")}
+
+  FRIENDLY_NAME = "Workflows Credential"
+
+  COMMON_ATTRIBUTES = [
+    {
+      :component  => 'text-field',
+      :label      => N_('Name'),
+      :helperText => N_('Name of this credential'),
+      :name       => 'name',
+      :id         => 'name'
+    },
+    {
+      :component  => 'text-field',
+      :label      => N_('Reference'),
+      :helperText => N_('Unique reference for this credential'),
+      :name       => 'ems_ref',
+      :id         => 'ems_ref'
+    },
+    {
+      :component  => 'text-field',
+      :label      => N_('Username'),
+      :helperText => N_('Username for this credential'),
+      :name       => 'userid',
+      :id         => 'userid'
+    },
+    {
+      :component  => 'password-field',
+      :label      => N_('Password'),
+      :helperText => N_('Password for this credential'),
+      :name       => 'password',
+      :id         => 'password',
+      :type       => 'password'
+    },
+    {
+      :component      => 'password-field',
+      :label          => N_('Private key'),
+      :helperText     => N_('RSA or DSA private key to be used instead of password'),
+      :componentClass => 'textarea',
+      :name           => 'auth_key',
+      :id             => 'auth_key',
+      :type           => 'password'
+    },
+    {
+      :component  => 'password-field',
+      :label      => N_('Private key passphrase'),
+      :helperText => N_('Passphrase to unlock SSH private key if encrypted'),
+      :name       => 'auth_key_password',
+      :id         => 'auth_key_password',
+      :maxLength  => 1024,
+      :type       => 'password'
+    }
+  ].freeze
+
+  API_ATTRIBUTES = COMMON_ATTRIBUTES
```

Comment:
NOTE we don't have any "EXTRA_ATTRIBUTES" so it seemed silly to define it then not use it.
The only reference I found to it other than `API_ATTRIBUTES = COMMON+EXTRA` was in AWX/Tower in the refresh process `inventory_object.options = inventory_object.type.constantize::EXTRA_ATTRIBUTES.keys.each_with_object({}) do |k, h|` so we don't seem to need it here.